scripts/install_third_party.sh

```diff
@@ -110,3 +110,10 @@ if [ ! "$NO_SKULPT" -a ! -d "$THIRD_PARTY_DIR/static/skulpt-0.10.0" ]; then
   mkdir -p $THIRD_PARTY_DIR/static/skulpt-0.10.0
   cp -r $TOOLS_DIR/skulpt-0.10.0/skulpt/dist/* $THIRD_PARTY_DIR/static/skulpt-0.10.0
 fi
+
+echo Checking whether node-jscs dependencies are installed
+if [ ! -d "$NODE_MODULE_DIR/jscs" ]; then
+  echo installing node-jscs
+  cd $COMMON_DIR
```

Comment:
sorry,removed that command. I though since ```if block``` before this we were in OPPIA_DIR so calling directly might install inside oppia directory.